funsite/static/funsite/css/header.css

```diff
@@ -41,6 +41,21 @@
     font-family: 'Raleway', sans-serif;
 }
 
+#top-menu span.fun-logo {
+    background-image: url('../images/logos/fun61.png');
```

Comment:
This is not compatible with the most recent FUN_BIG_LOGO_RELATIVE_PATH setting :-/
Could we solve the problem with jpg images instead of png images?